hooks/boot/06-postgresql-upgrade-extensions.rb

```diff
@@ -12,6 +12,9 @@ def os_needs_postgresql_upgrade?
   def postgresql_upgrade(new_version)
     logger.notice("Performing upgrade of PostgreSQL to #{new_version}")
 
+    start_services(['postgresql'])
+    (_name, _owner, _enconding, collate, ctype, _privileges) = `runuser -l postgres -c 'psql -lt | grep -E "^\s+postgres"'`.chomp.split('|').map(&:strip)
```

Comment:
Can we use `execute_as!` here?

Reply:
not without refactoring, as it currently does not return stdout

Reply:
I could get away without the `|grep` and filter in Ruby, but I didn't feel like I want to open up the refactoring pot right now and the code above is copied from the olde el7-pg12 upgrade :)